include/tcframe/generator/TestCaseGenerator.hpp

```diff
@@ -61,7 +61,9 @@ class TestCaseGenerator {
             applyInput(testCase);
             verifyInput(testCase);
             generateInput(testCase, inputFilename, config);
-            evaluateAndApplyOutput(testCase, inputFilename, outputFilename, config);
+            if (config.generateOutput()) {
```

Comment:
This *definitely* needs unit test(s) in `TestCaseGeneratorTests`.

You need to make sure that `evaluate()` is not called (`Times(0)`). See https://github.com/tcframe/tcframe/blob/v1.1.0/test/tcframe/grader/TestCaseGraderTests.cpp#L100 for reference